pkg/gardenlet/controller/backupentry/backupentry/reconciler.go

```diff
@@ -81,6 +81,11 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	log := logf.FromContext(ctx)
 
+	gardenCtx, cancel := context.WithTimeout(ctx, 11*time.Minute)
+	defer cancel()
+	seedCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
+	defer cancel()
```

Comment:
> Let's drop the WaitUntilObjectReadyWithHealthFunction in this controller in favor of proper watches to eliminate these insanely high timeouts.
In my opinion it will be better to drop `WaitUntilObjectReadyWithHealthFunction` in a different PR.

Reply:
https://github.com/gardener/gardener/issues/7517